bin/rails

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env ruby1.9.1
```

Comment:
I've just noticed that this should probably be `/usr/bin/env ruby`, as per how `bin/string` got generated on your system. I wonder if this is because I'm using `ruby1.9.1` via apt alternatives. I'd be in favour of a commit that updates all of these to `#!/usr/bin/env ruby`.